pkg/utils/activity/config.go

```diff
@@ -9,6 +9,7 @@ import (
 type Config struct {
 	Enabled           bool          `split_words:"true" default:"false"`
 	Topic             string        `split_words:"true"`
+	Network           string        `split_words:"true"`
```

Comment:
You could make this a `Network` instead of a `string` then instead of using `ParseNetwork` you could implement the `Decode` interface and also add `Validate` to the `Network` type so that confire will automatically parse and validate the network for you -- it might eliminate some boilerplate to have the type be self describing. We do the same thing with the LogLevel type. 

https://github.com/rotationalio/confire#decoder-interface